packages/ketcher-react/src/script/ui/state/modal/form.js

```diff
@@ -19,6 +19,7 @@ import { initSdata, sdataReducer } from './sdata'
 
 export const formsState = {
   // TODO: create from schema.{smth}.defaultValue
+  // TODO: change validation method
```

Comment:
What do you plan to do  here?

Reply:
When discussing forms' validation with Pavel Sidorenko, we noticed that instead of proper validation, we pass 'valid:true', which leads to the 'OK' button being constantly enabled. As such, we agreed to add a TODO to change our validation process.